CMakeLists.txt

```diff
@@ -349,6 +349,7 @@ if(NOT CMAKE_BUILD_TYPE)
  mark_as_advanced(CMAKE_BUILD_TYPE)
 endif(NOT CMAKE_BUILD_TYPE)
 message(STATUS "Build type: " ${CMAKE_BUILD_TYPE})
+set(CMAKE_CXX_FLAGS_RELEASE "-O3 -march=native -DNDEBUG" CACHE STRING " The release flags " FORCE)
```

Comment:
Good point, but there is a problem with Mac OSX. 

At least for gcc the default system assembler do not support the instructions generated by the compiler. To select the gcc assembler one has to use "-Wa,-q -march=native".

I know this is OSX being broken and we maybe should not care. But this will potentially generate more "can not compile triqs" issue reports.

(Note that "-march=native" is not enough to get AVX-FMA instructions since it changes the floating point round off error.)